var/spack/repos/builtin/packages/py-update-checker/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyUpdateChecker(PythonPackage):
+    """A python module that will check for package updates."""
+
+    homepage = "https://github.com/bboe/update_checker"
+    url      = "https://pypi.io/packages/source/u/update_checker/update_checker-0.18.0.tar.gz"
+
+    version('0.18.0', sha256='6a2d45bb4ac585884a6b03f9eade9161cedd9e8111545141e9aa9058932acb13')
+    version('0.17',   sha256='2def8db7f63bd45c7d19df5df570f3f3dfeb1a1f050869d7036529295db10e62')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-pytest',     type='test')
+    depends_on('py-black',      type='test')
+    depends_on('py-flake8',     type='test')
```

Comment:
Are these two needed for testing or just for linting? Like, when you run the unit tests, are these used in any way?

Reply:
According to setup.py, they're used for linting. So, I guess I can either remove them or create a lint variant

Reply:
I would remove them. Even if the code doesn't lint correctly, it will still run correctly.